### PR TITLE
Created script to generate benchmark data

### PR DESCRIPTION
**Pull request by Carson21**
Also fixed a typo in the query.sql file and state.csv where Ethnicity was spelled wrong which resulted in compare.py failing when running the benchmark.

### CODE REVIEW

File: CDC-Data-Reconciliation-Backend/query.sql [hunk 1]
```diff
@@ -33,7 +33,7 @@ Public_health_case.local_id AS CaseID
 	   else '9' end as Race
        ,case when Person.ethnic_group_ind ='2135-2' then '1'
 	   when Person.ethnic_group_ind ='2186-5' then '2'
-	   else '9' end as Ethnicty
+	   else '9' end as Ethnicity
```
Comment (waffy1901) [inline CDC-Data-Reconciliation-Backend/query.sql:36]: Great catch here.

File: CDC-Data-Reconciliation-Backend/state.csv [hunk 2]
```diff
@@ -1,4 +1,4 @@
-add_time,CaseID,CountyReporting,EventCode,EventName,MMWRYear,MMWRWeek,CaseClassStatus,Sex,BirthDate,Age,AgeType,Race,Ethnicty
+add_time,CaseID,CountyReporting,EventCode,EventName,MMWRYear,MMWRWeek,CaseClassStatus,Sex,BirthDate,Age,AgeType,Race,Ethnicity
```
Comment (waffy1901) [inline CDC-Data-Reconciliation-Backend/state.csv:1]: Phenomenal attention to detail!

**Review by waffy1901 — APPROVED** [review]
Great idea to utilize a Python script to generate csv files for the cdc and states with arbitrary data. These are all very important changes - great work!

**Review by john-baute — COMMENTED** [review]
LGTM

**Pull request merged**